src/ConnectedMode/UI/Resources/UiResources.resx

```diff
@@ -357,6 +357,12 @@
   <data name="FetchingBindingStatusText" xml:space="preserve">
     <value>Fetching binding status...</value>
   </data>
+  <data name="CheckingForSharedBindingFailedText" xml:space="preserve">
+    <value>Checking for shared configuration...</value>
```

Comment:
The resources should be swapped. You put the checking for shared configuration in the resource with the "Failed" key.